src/coreclr/src/vm/peimage.h

```diff
@@ -119,11 +119,9 @@ class PEImage
 
     };
 
-    static PTR_PEImage FindById(UINT64 uStreamAsmId, DWORD dwModuleId);
     static PTR_PEImage FindByPath(LPCWSTR pPath,
-                                  BOOL isInBundle = TRUE);
-    static PTR_PEImage FindByShortPath(LPCWSTR pPath);
-    static PTR_PEImage FindByLongPath(LPCWSTR pPath);
+                                  BOOL isInBundle);
```

Comment:
This default value is not actually used anywhere today. If it got ever used, there is very high chance that it would not be right.